### DISCUSSION TITLE
Falsely reporting that there isn’t enough free space

### DISCUSSION THREAD

**Discussion by disenchant** [iterate-ch/cyberduck#14938] [Q&A]
We've got a Nextcloud instance on which users have a quota limit of 0 Bytes which ensures that they can't place files in their home folders but these quotas don't apply to folders shared with them. Nevertheless Mountain Duck seems to think based on the quota (at least that's what I assume) that it is for everything on the share. This leads to errors like the following when trying to copy or create any files.

![2023-07-26 at 11 00 51](https://github.com/iterate-ch/cyberduck/assets/686611/b0011311-0aff-400d-b065-8d648587146b)

Tested on macOS 13.4.1 (22F82) with Mountain Duck version 4.14.0 (21323)

**Comment by tatze96** [comment]
Cyberduck and Mountain Duck request the global quota, not the quota from each separate folder. To get the correct quota allocated, you'll have to create individual bookmarks for the home folder and the shared folders.
